ontology/yaml/resources/TRANSPORT/entity_types/ABSTRACT.yaml

```diff
@@ -23,3 +23,16 @@ SS:
   - failed_alarm
   - lost_power_alarm
 
+EPM:
```

Comment:
What is the floor status used for? Is it important enough to make a required point?

Reply:
just for information on which floor the elevator is on